lua/lualine/init.lua

```diff
@@ -260,8 +261,8 @@ local function set_statusline()
   if next(config.sections) ~= nil or next(config.inactive_sections) ~= nil then
     vim.o.statusline = '%!v:lua.require\'lualine\'.statusline()'
     vim.api.nvim_exec([[
-    autocmd lualine WinLeave,BufLeave * lua vim.wo.statusline=require'lualine'.statusline()
-    autocmd lualine WinEnter,BufEnter * set statusline<
+    autocmd lualine VimResized, WinLeave,BufLeave * lua vim.wo.statusline=require'lualine'.statusline()
```

Comment:
I'm actually not sure what this change does . Will having a separate auto command for VimResized executing `:redrawsratus` have same effect?

Reply:
> I'm actually not sure what this change does . 
Well obviously it redraws the statusline on `VimResized` 😆 I think it's useful, at least for me. If you maximize neovim window you probably want the shortened path to be displayed as big as possible again

>Will having a separate auto command for VimResized executing `:redrawstatus` have same effect?
yes, that is better imo.